src/drivers/ll40ls/ll40ls.cpp

```diff
@@ -348,9 +350,12 @@ int
 ll40ls_main(int argc, char *argv[])
 {
 	int ch;
+	int myoptind = 1;
+	const char *myoptarg = NULL;
 	enum LL40LS_BUS busid = LL40LS_BUS_I2C_ALL;
+	enum Rotation rotation = ROTATION_NONE;
 
-	while ((ch = getopt(argc, argv, "XI")) != EOF) {
+	while ((ch = px4_getopt(argc, argv, "IXR:", &myoptind, &myoptarg)) != EOF) {
```

Comment:
@TSC21 I started here with the uppercase R and then did it the same way for all the drivers... Also, an external company already used a capital R, that's why I sticked with it.
If you don't mind I would keep it but I'm flexible.

Reply:
Just cause I saw some places with lowercase and others with upper. Just wanted to keep the consistency.

Reply:
I guess that's ok as long as the capital R for the rotation is consistent.